### ISSUE TITLE
Error during game play

### ISSUE THREAD

**Issue by ShaneGH** [CorsixTH/CorsixTH#1265]
#### Describe the issue
Error in game. Game recovers well, but I thought you guys might want to know about it.

#### Steps to Reproduce
1. *Load save game attached to this ticket*

2. The error should happen as soon as you load. Note: possibly something to do with an earthquake that just happened and destroyed some clinics.

#### Save Game
https://drive.google.com/open?id=0B_U6MBBsnuUpa1VkaTFiNlFNQzg

#### System Information
**CorsixTH Version:** 0.6

**Operating System:** Windows 10

**Theme Hospital Version:** Not sure

#### Gamelog.txt
```
Created game with savegame version 111.
Savegame version is 111 (v0.60), originally it was 111 (v0.60)
Warning: Trying to remove nonexistant room build callback (function: 000002753E7DCE90) from humanoid (table: 00000275400D0A10).
Warning: Trying to remove nonexistant room remove callback (function: 000002753DBD4360) from humanoid (table: 00000275400D0A10).
Warning: Trying to remove nonexistant room build callback (function: 000002753EDEF2A0) from humanoid (table: 00000275400D0A10).
Warning: Trying to remove nonexistant room remove callback (function: 000002753D7537A0) from humanoid (table: 00000275400D0A10).
Warning: Trying to remove nonexistant room build callback (function: 000002753EDE5160) from humanoid (table: 000002753DFE2600).
Warning: Trying to remove nonexistant room remove callback (function: 000002753D7553F0) from humanoid (table: 000002753DFE2600).
Warning: Trying to remove nonexistant room build callback (function: 000002753EDF8E40) from humanoid (table: 000002753DD57930).
Warning: Trying to remove nonexistant room remove callback (function: 0000027540E9E8D0) from humanoid (table: 000002753DD57930).
Warning: Trying to remove nonexistant room build callback (function: 000002753828CE50) from humanoid (table: 000002753E961320).
Warning: Trying to remove nonexistant room remove callback (function: 000002753DBDCBB0) from humanoid (table: 000002753E961320).
Warning: Trying to remove nonexistant room build callback (function: 000002753E7DA790) from humanoid (table: 000002753EE53CA0).
Warning: Trying to remove nonexistant room remove callback (function: 0000027543FDC1E0) from humanoid (table: 000002753EE53CA0).
Warning: Trying to remove nonexistant room build callback (function: 000002753EDFA100) from humanoid (table: 00000275400CA990).
Warning: Trying to remove nonexistant room remove callback (function: 0000027543E1EC30) from humanoid (table: 00000275400CA990).
Warning: Trying to remove nonexistant room build callback (function: 000002753EDF1DC0) from humanoid (table: 000002753FB0ED60).
Warning: Trying to remove nonexistant room remove callback (function: 0000027544053B30) from humanoid (table: 000002753FB0ED60).
Warning: Trying to remove nonexistant room build callback (function: 000002753E7F2DF0) from humanoid (table: 0000027546F0E210).
Warning: Trying to remove nonexistant room remove callback (function: 0000027540BF8760) from humanoid (table: 0000027546F0E210).
Warning: Trying to remove nonexistant room build callback (function: 0000027538298C10) from humanoid (table: 000002753E134DE0).
Warning: Trying to remove nonexistant room remove callback (function: 000002753DBE00F0) from humanoid (table: 000002753E134DE0).
Warning: Trying to remove nonexistant room build callback (function: 000002753E7E6610) from humanoid (table: 000002754413E990).
Warning: Trying to remove nonexistant room remove callback (function: 00000275473081D0) from humanoid (table: 000002754413E990).
Warning: Trying to remove nonexistant room build callback (function: 000002753E7E32B0) from humanoid (table: 000002753F251F20).
Warning: Trying to remove nonexistant room remove callback (function: 0000027543FD4E60) from humanoid (table: 000002753F251F20).
Savegame version is 111 (v0.60), originally it was 111 (v0.60)
Warning: Trying to remove nonexistant room build callback (function: 00000224B6B159A0) from humanoid (table: 00000224B6872A50).
Warning: Trying to remove nonexistant room remove callback (function: 00000224B76AC820) from humanoid (table: 00000224B6872A50).
Warning: Trying to remove nonexistant room build callback (function: 00000224B9226970) from humanoid (table: 00000224BBA052F0).
Warning: Trying to remove nonexistant room remove callback (function: 00000224B7E32E80) from humanoid (table: 00000224BBA052F0).
Warning: Trying to remove nonexistant room build callback (function: 00000224B6B1A4A0) from humanoid (table: 00000224B74F53E0).
Warning: Trying to remove nonexistant room remove callback (function: 00000224B82886A0) from humanoid (table: 00000224B74F53E0).
Warning: Trying to remove nonexistant room build callback (function: 00000224B9225DD0) from humanoid (table: 00000224B8914220).
Warning: Trying to remove nonexistant room remove callback (function: 00000224B76AD420) from humanoid (table: 00000224B8914220).
Warning: Trying to remove nonexistant room build callback (function: 00000224B922C370) from humanoid (table: 00000224BBE207F0).
Warning: Trying to remove nonexistant room remove callback (function: 00000224B7E2D120) from humanoid (table: 00000224BBE207F0).
Savegame version is 111 (v0.60), originally it was 111 (v0.60)
Warning: Trying to remove nonexistant room build callback (function: 00000224B6B20FE0) from humanoid (table: 00000224B78C2DB0).
Warning: Trying to remove nonexistant room remove callback (function: 00000224B7E32CA0) from humanoid (table: 00000224B78C2DB0).
Savegame version is 111 (v0.60), originally it was 111 (v0.60)
Savegame version is 111 (v0.60), originally it was 111 (v0.60)
Warning: Trying to remove nonexistant room build callback (function: 00000224B6239E20) from humanoid (table: 00000224B90AA870).
Warning: Trying to remove nonexistant room remove callback (function: 00000224BCD31370) from humanoid (table: 00000224B90AA870).
Savegame version is 111 (v0.60), originally it was 111 (v0.60)
Error in timer handler: 
C:\Program Files\CorsixTH\Lua\world.lua:1804: bad argument #2 to 'findObject' (number expected, got nil)
stack traceback:
	[C]: in method 'findObject'
	C:\Program Files\CorsixTH\Lua\world.lua:1804: in method 'findObjectNear'
	C:\Program Files\CorsixTH\Lua\entities\staff.lua:186: in method 'tick'
	C:\Program Files\CorsixTH\Lua\world.lua:1075: in method 'onTick'
	C:\Program Files\CorsixTH\Lua\app.lua:1000: in function <C:\Program Files\CorsixTH\Lua\app.lua:997>
	(...tail calls...)
	C:\Program Files\CorsixTH\Lua\app.lua:901: in function <C:\Program Files\CorsixTH\Lua\app.lua:896>

```

**Comment by TheCycoONE** [comment]
Duplicate of #1195

**Comment by mugmuggy** [comment]
Just covering off one thing I noticed when the room crasheds, a bin was left in the room.
Room:crashRoom 
`for object, _ in pairs(self.world:findAllObjectsNear(fx, fy)) do`
findAllObjectsNear appears only to return the first object on a tile at least in this build. Bin was under the xray viewer and was left behind. I put a bin and radiator under the xray viewer and that left 2 objects in the room.

**Comment by TheCycoONE** [comment]
Thanks @mugmuggy I know how to fix that.

**Comment by mugmuggy** [comment]
My last commit into #1270 was to fix the 2 objects 1 tile issue, that I broke when fixing the objects on litter. So that will prevent this issue from occuring in future builds once merged

**Comment by TheCycoONE** [comment]
Ah, I see

**Comment by mugmuggy** [comment]
Fixed with #1451
